src/lib/components/documents/ResultsList.svelte

```diff
@@ -40,10 +103,32 @@
       <p>{$_("noDocuments.queryNoResults")}</p>
     </Empty>
   {/each}
+  <div class="end" bind:this={end}>
+    {#if next}
+      <Button disabled={loading} on:click={(e) => load(new URL(next))}>
+        {#if loading}
+          Loading ...
+        {:else}
+          Load more
+        {/if}
+      </Button>
+    {/if}
+  </div>
```

Comment:
I think this may be all we need for cleaner styling:

```suggestion
  <Flex justify="center" bind:this={end}>
    {#if next}
      <Button mode="ghost" disabled={loading} on:click={(e) => load(new URL(next))}>
        {#if loading}
          Loading…
        {:else}
          Load more
        {/if}
      </Button>
    {/if}
  </Flex>
```

Reply:
Needed to keep an element bindable here, so did the flexing the old fashioned way.